### PR TITLE
Create the data model

### PR DESCRIPTION
**Pull request by yozhgoor**

### CODE REVIEW

File: examples/yozhgoor/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                website: "".to_string(),
+                description: "As an ambulance driver".to_string(),
+                technologies: vec!["".to_string()],
+                period: "2018".to_string(),
+            },
+            Job {
+                company: "Charly's Company".to_string(),
+                website: "https://charlyscompany.be".to_string(),
+                description: "as a LMV (Light Medical Vehicle) driver".to_string(),
+                technologies: vec!["".to_string()],
```
Comment (yozhgoor) [inline examples/yozhgoor/src/lib.rs:47]: I can change this Vec<String> in a option<Vec<String>>, but in this case I need to switch all the technologies field or just the one in job (Contribution have these tags in most cases I think, so Candidate will have these tags in most cases)

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -14,17 +14,7 @@ jobs:
...
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.rustup
-            ~/.cargo/registry
-            ~/.cargo/bin
-            target
-          key: ${{ github.event.issue.number }}-${{ runner.os }}-pr
-          restore-keys: main
+      - uses: Swatinem/rust-cache@v1
```
Comment (cecton) [inline .github/workflows/pr.yml:17]: You need to sync the change with the main.yml

File: examples/yozhgoor/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                "Reddit".to_string(),
+                "https://www.reddit.com/user/yozhgoor".to_string(),
+            ),
+        ],
+        jobs: vec![
+            Job {
+                company: "CJP Ambulances du centre".to_string(),
+                website: "".to_string(),
+                description: "As an ambulance driver".to_string(),
+                tech: vec!["".to_string()],
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:40]: Why an empty string? Why not just an empty vec?

File: examples/yozhgoor/src/lib.rs [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+                website: "https://crates.io/crates/wasm-run".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "CLI".to_string(),
+                    "Library".to_string(),
+                    "Script".to_string(),
+                ],
+                description: "Build tool that replaces cargo run to build WASM projects.
+                    Just like webpack, wasm-run offers a great deal of customization."
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:78]: That's not how you do multiline strings in Rust. I let you ddg that

File: examples/yozhgoor/src/lib.rs [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+                    "CSS".to_string(),
+                    "Embedded".to_string(),
+                    "PWA".to_string(),
+                ],
+                description: "Third-I, the only camera that replicates your hearing
+                    and your point of view in the first person."
+                    .to_string(),
+            },
+            Contribution {
+                project: "Wasm-run".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:68]: It's called "wasm-run" not "Wasm-run"

File: examples/yozhgoor/src/lib.rs [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+                website: "https://crates.io/crates/wasm-run".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "CLI".to_string(),
+                    "Library".to_string(),
+                    "Script".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:75]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+                    .to_string(),
+            },
+            Contribution {
+                project: "Wasm-run".to_string(),
+                website: "https://crates.io/crates/wasm-run".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "CLI".to_string(),
+                    "Library".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:74]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+                description: "Third-I, the only camera that replicates your hearing
+                    and your point of view in the first person."
+                    .to_string(),
+            },
+            Contribution {
+                project: "Wasm-run".to_string(),
+                website: "https://crates.io/crates/wasm-run".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:72]: It's called "WASM", not "Wasm"

File: examples/yozhgoor/src/lib.rs [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+                    "CLI".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:73]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+                project: "Yewprint".to_string(),
+                website: "https://yewprint.rm.rs/".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "Yew".to_string(),
+                    "Library".to_string(),
+                    "HTML".to_string(),
+                    "CSS".to_string(),
+                    "Script".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:91]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+            Contribution {
+                project: "Yewprint".to_string(),
+                website: "https://yewprint.rm.rs/".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "Yew".to_string(),
+                    "Library".to_string(),
+                    "HTML".to_string(),
+                    "CSS".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:90]: We don't really do CSS in that project...

File: examples/yozhgoor/src/lib.rs [hunk 12]
```diff
@@ -0,0 +1,151 @@
...
+            },
+            Contribution {
+                project: "Yewprint".to_string(),
+                website: "https://yewprint.rm.rs/".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "Yew".to_string(),
+                    "Library".to_string(),
+                    "HTML".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:89]: We don't do HTML either

File: examples/yozhgoor/src/lib.rs [hunk 13]
```diff
@@ -0,0 +1,151 @@
...
+                    .to_string(),
+            },
+            Contribution {
+                project: "Yewprint".to_string(),
+                website: "https://yewprint.rm.rs/".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "Yew".to_string(),
+                    "Library".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:88]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 14]
```diff
@@ -0,0 +1,151 @@
...
+                description: "Build tool that replaces cargo run to build WASM projects.
+                    Just like webpack, wasm-run offers a great deal of customization."
+                    .to_string(),
+            },
+            Contribution {
+                project: "Yewprint".to_string(),
+                website: "https://yewprint.rm.rs/".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:86]: WASM, not Wasm

File: examples/yozhgoor/src/lib.rs [hunk 15]
```diff
@@ -0,0 +1,151 @@
...
+            Contribution {
+                project: "Yewprint-playground".to_string(),
+                website: "https://github.com/yozhgoor/yewprint-playground".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "Wasm".to_string(),
+                    "Yew".to_string(),
+                    "HTML".to_string(),
+                    "CSS".to_string(),
+                    "Netlify".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:106]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 16]
```diff
@@ -0,0 +1,151 @@
...
+                description: "a Yew and yewprint playground using wasm-run".to_string(),
+            },
+            Contribution {
+                project: "Rusty-snake".to_string(),
+                website: "https://github.com/yozhgoor/rusty-snake".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "CLI".to_string(),
+                    "Game".to_string(),
+                    "Design".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:117]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 17]
```diff
@@ -0,0 +1,151 @@
...
+                ],
+                description: "a Yew and yewprint playground using wasm-run".to_string(),
+            },
+            Contribution {
+                project: "Rusty-snake".to_string(),
+                website: "https://github.com/yozhgoor/rusty-snake".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "CLI".to_string(),
+                    "Game".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:116]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 18]
```diff
@@ -0,0 +1,151 @@
...
+                    "Netlify".to_string(),
+                ],
+                description: "a Yew and yewprint playground using wasm-run".to_string(),
+            },
+            Contribution {
+                project: "Rusty-snake".to_string(),
+                website: "https://github.com/yozhgoor/rusty-snake".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "CLI".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:115]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 19]
```diff
@@ -0,0 +1,151 @@
...
+                    "Netlify".to_string(),
+                ],
+                description: "My vision of the Rust Programming Language,
+                    as an introduction to Rust"
+                    .to_string(),
+            },
+        ],
+        contract_type: ContractType::Contractor,
+        availability: Availability::NotAvailable,
+        joined_date: Instant::now(),
```
[the remark this answers is not in the document]
Comment (cecton) [reply examples/yozhgoor/src/lib.rs:148]: We should probably change this indeed

File: examples/yozhgoor/src/lib.rs [hunk 20]
```diff
@@ -0,0 +1,151 @@
...
+                ],
+                description: "Snake, the game in your terminal".to_string(),
+            },
+            Contribution {
+                project: "Introduction à Rust".to_string(),
+                website: "https://intro-to-rust.netlify.app".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "MdBook".to_string(),
+                    "Tutorial".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:138]: Not a tech

File: examples/yozhgoor/src/lib.rs [hunk 21]
```diff
@@ -0,0 +1,151 @@
...
+                    "Design".to_string(),
+                ],
+                description: "Snake, the game in your terminal".to_string(),
+            },
+            Contribution {
+                project: "Introduction à Rust".to_string(),
+                website: "https://intro-to-rust.netlify.app".to_string(),
+                tech: vec![
+                    "Rust".to_string(),
+                    "MdBook".to_string(),
```
Comment (cecton) [inline examples/yozhgoor/src/lib.rs:137]: Not a tech

File: examples/yozhgoor/src/lib.rs (hunk 20 above)
Comment (cecton) [reply examples/yozhgoor/src/lib.rs:138]: Can you imagine recruiter looking for "MdBook"? lol

File: src/main.rs [hunk 22]
```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::unnecessary_wraps)]
```
Comment (cecton) [inline src/main.rs:1]: I love how you are doing the exact opposite of what your commit says :joy: 

>  Deny a clippy lint 

```
 #![allow(clippy::unnecessary_wraps)]
```

Comment (yozhgoor) [reply src/main.rs:1]: Yup miss steak

**Pull request merged**